components/automate-gateway/gateway/services.go

```diff
@@ -139,7 +144,7 @@ func (s *Server) RegisterGRPCServices(grpcServer *grpc.Server) error {
 		return errors.Wrap(err, "create client for authzV2 service")
 	}
 	pb_authz.RegisterAuthorizationServer(grpcServer,
-		handler.NewAuthzServer(authzClient, authzV2Client, s.authorizer))
+		handler.NewAuthzServer(authzClient, s.authorizer))
```

Comment:
This line is the only substantive change in this file, removing a no-longer-needed argument.
All the remaining changes in this file are making lines shorter.